packages/core/src/lib/markdown-it/patches/custom-component/index.js

```diff
@@ -0,0 +1,11 @@
+/*
+ * https://github.com/vuepress/vuepress-next/tree/main/packages/%40vuepress/markdown/src/plugins/customComponentPlugins
+ *
+ * The files in this folder, custom-component, are adapted from the above link to fit MarkBind's needs. 
+ *
+ * It helps us to define Vue custom components (or unknown components) as block-level elements during markdown-it parsing.
```

Comment:
```suggestion
 * It helps us to define Vue custom components (or unknown components) as block or inline level elements during markdown-it parsing.
```

could use a mention of the "special tags" patch as well (`<script/style>`)